backend/main.go

```diff
@@ -359,7 +359,9 @@ func main() {
 	r.Use(httplog.RequestLogger(errorLogger))
 	compressor := middleware.NewCompressor(5, "application/json")
 	compressor.SetEncoder("br", func(w io.Writer, level int) io.Writer {
-		return brotli.NewWriterLevel(w, level)
+		return cbrotli.NewWriter(w, cbrotli.WriterOptions{
+			Quality: level,
```

Comment:
5 seems reasonable, though in the future could look at lowering the level further 
<img width="482" alt="Screenshot 2023-11-17 at 9 43 50 AM" src="https://github.com/highlight/highlight/assets/1351531/1272668a-6ea1-405a-befd-40200a660e7d">

Reply:
👍 agree that level 5 is probably overkill
I don't think this is a big deal atm because private graph uses a fraction of the CPU / mem out of all our services, but if this shows up in profiling would be good to do
Incidentally was looking at similar performance benchmarks and I think it makes sense to reduce the brotli level from 9 to 5 for our long-term session storage #7164